src/main/java-templates/org/postgresql/Driver.java

```diff
@@ -411,7 +411,7 @@ public Connection getResult(long timeout) throws SQLException {
      * @throws SQLException if the connection could not be made
      */
     private static Connection makeConnection(String url, Properties props) throws SQLException {
-        return new @JDBCCONNECTCLASS@(hostSpecs(props),
+        return new ${JDBCCONNECTCLASS}(hostSpecs(props),
```

Comment:
@mkarg , can you please raise an issue with the change you have in mind?

I think we'd better move to a proper discussion thread.

PS. I have no problem with adding version-specific class names provided universal `org.postgresql.Driver` is still working.